src/i-client-config.ts

```diff
@@ -0,0 +1,102 @@
+import { Flag, IAssignmentLogger, IAsyncStore } from '@eppo/js-client-sdk-common';
+
+import { ServingStoreUpdateStrategy } from './isolatable-hybrid.store';
+
+/**
+ * Configuration used for initializing the Eppo client
+ * @public
+ */
+export interface IClientConfig {
```

Comment:
Thanks for pulling this out into it's own file, much cleaner 🧼

Reply:
🧹